.gitignore

```diff
@@ -3,4 +3,4 @@ npm-debug.log
 build
 generated
 .antwar
-
+.vscode
```

Comment:
Absolutely. Sorry about that. Was meant to be a nod/joke to @TheLarkInn. I can cancel and resubmit though.